### ISSUE TITLE
Élan Namadgi - flat batteries

### ISSUE THREAD

**Issue by DaveHolt** [CanberraOceanRacingClub/namadgi3#222]
It looks like all house and starter batteries are flat on Élan Namadgi. Or there is another problem preventing any 12 volt power to any appliance or device on board. DaveH will follow up with BME and advise DBY Boat sales.

**Comment by DaveHolt** [comment]
BME will remove all the batteries and recharge in a controlled manner in their workshop. Following that we will address why they went flat. Solar panels not working or other problem.

**Comment by smr547** [comment]
Do we have an update on this yet? Speaking with Greg Hall today he said Chris from BME told him that the batteries were dead and would have to be replaced.

**Comment by DaveHolt** [comment]
Maybe Greg knows more than me. Daniel said they would remove the batteries and recharge them slowly in a controlled way.

**Comment by smr547** [comment]
That sounds like the best plan. I don't know why Chris of BME would contact Greg. Best you deal direct with Dan and we will await further updates. Cheers Steven

**Comment by DaveHolt** [comment]
Hi,
I spoke to Chris at BME he will try to slowly recharge the batteries in the workshop but suspects that they are beyond recovery. He has sent a first cut quote to replace all 5 x 110 Ah  AGM batteries with identical units for around $2600 incl GST. I've asked him to consider some options e.g smaller starter with same CCA; maybe only 2-3 house batteries of a cheaper construction. The boat came originally with 1 start and 2 house batteries. We had them replaced with bigger batteries and two extras in 2012 or 13 I think. 

Second: they need to discover why the batteries went flat with 2 solar panels attached. I get the solar panels checked every annual service.

**Comment by DaveHolt** [comment]
Chris has come back with a much cheaper option for similar spec 100Ah Federal sealed maintenance free batteries each $286. So we could have it going again with 3 batteries for about a thousand or $1300ish for 4 batteries or $1500ish for 5. Plus a cost to sort the solar panels recharging. I'm not in favour of spending heaps on a boat that's for sale. @bullswool

**Comment by DaveHolt** [comment]
The Committee has agreed on a solution.
I have asked Chris at BME to order and replace 1 starter and 2 house 110 A/h AGM batteries. Cost each $525 = $1575 plus labour
And Reset the Xantrex battery meter
And resolve why the solar panels didn't keep them charged.
With labour this will be about $2000

For reference BME advise that there were 4 house AGM 110 A/h batteries and 1 starter/anchor AGM 110 A/h.

**Comment by DeeTaylor87** [comment]
Can this be checked at the working bee? I've made an Elan Working Bee label for it...

**Comment by DaveHolt** [comment]
3 batteries have been replaced. BME still have not investigated the solar panels. I have complained. 
It would be good for someone at the working bee to re-calibrate the Xantrex.

**Comment by smr547** [comment]
Hi Dave, 

We will recalibrate Xantrex *and* do a battery audit:
* check open circuit voltages (deducing SOC)
* measure charging/discharging current in "boat off" state (to see if there is still a problem)
* check charging current
  * main motor
  * genset (don't think we can debug the 10 min shutdown problem)
  * shore power
  * Solar (if available)

**Comment by DaveHolt** [comment]
Hi All, it sounds like BME have solved the solar panel problem. The following is from Chris at BME:

I have solved the issue with the solar system.I traced the solar regulator wiring back to a fuse that had blown at some point. The system is now working.

I was also able to track down the cause of the engine indicator light being on all the time and the slow discharge on the batteries. I found that a cable had been rubbing on one of the battery switch terminals and had worn through the insulation. This meant when you turned off the engine battery switch you were actually not doing anything and the engine could still be started. I have repaired the cable and moved it out of harms way.

It is highly likely that the problem I just spoke of above was the cause of the batteries going flat as the engine controls and gauges were always powered up causing the engine battery to go flat. In conjunction with the emergency parallel being left on this would have run all the batteries dead.

Let's leave this issue open until it can be checked at the working bee.
